src/node/setupServer.ts

```diff
@@ -62,6 +69,32 @@ export const setupServer = (...handlers: RequestHandler<any, any>[]) => {
       })
     },
 
+    /**
+     * Prepends given request handlers to the list of existing handlers.
+     */
+    use(...handlers: RequestHandlersList) {
+      currentHandlers.unshift(...handlers)
+    },
+
+    /**
+     * Marks all request handlers that respond using `res.once()` as unused.
+     */
+    restoreHandlers() {
+      currentHandlers.forEach((handler) => {
```

Comment:
Both server and client code share the same logic in use/restoreHandlers/resetHandlers. Can this logic be abstracted into utility functions, so it's guaranteed to be the same for client and server?